open-api/rest-catalog-open-api.yaml

```diff
@@ -2800,6 +3062,58 @@ components:
           additionalProperties:
             type: string
 
+    PrePlanTableRequest:
+      type: object
+      required:
+        - select
+        - options
+      properties:
+        select:
```

Comment:
missing project(Schema)

Reply:
I think based on @rdblue suggestion we only wanted to have select. Is this still true?